core/src/main/resources/application.yml

```diff
@@ -73,7 +73,7 @@ feast:
     # See the following for options https://api.docs.feast.dev/grpc/feast.core.pb.html#KafkaSourceConfig
     options:
       topic: feast-features
-      bootstrapServers: localhost:9092
+      bootstrapServers: ${STREAM_HOST:localhost}:${STREAM_PORT:9092}
```

Comment:
You should be able to set this configuration without introducing new env vars. Have a look at Spring environmental variables. https://stackoverflow.com/a/48893053

Reply:
thanks, updated.